src/main/java/org/fcrepo/spec/testsuite/crud/HttpPut.java

```diff
@@ -355,36 +357,60 @@ public void nonRDFSourcePutDigestResponseHeaderVerification() {
     /**
      * 3.6.3-A
      */
-    @Test(groups = {"MAY"})
+    @Test(groups = {"MUST"})
```

Comment:
Why is this a must?  The spec says may.

Reply:
Looking at the spec I see the must in there:  I would suggest adding the following to the test description: 
```
On creation of an LDP-NR with HTTP PUT, implementations must create an associated LDP-RS describing that LDP-NR in the same way that it would when 3.5.1 Creating LDP-NRs with HTTP POST.
```

Reply:
Changing back to `MAY`.